packages/contracts/src/networks.ts

```diff
@@ -1109,6 +1141,10 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
     actionGasLimitBuffer: false,
     eip2028: true,
     actionTransactionBatching: true,
+    rollupStack: {
+      provider: 'None',
+      type: 'OP Stack',
+    },
```

Comment:
Blast is a modified OP stack chain. I don't know whether this impacts the cost estimate